src/intervaloptimisation.jl

```diff
@@ -77,22 +25,17 @@ its domain `dom`.
 
 ### Algorithm
 
-The solver finds the global minimum of the function `f` over the `Interval` or
-`IntervalBox` `dom` using the Moore-Skelboe algorithm. The method actually returns
-an *interval* containing the global minimum (resp. an *interval* containing the
-global maximum), as well as a list of boxes containing the minimisers
-(resp. maximisers); they can be obtained calling [`range_and_optimisers`](@ref).
-
-This function conservatively chooses the infimum (resp. supremum) over each interval.
+The solver finds the global minimum  and maximum of the function `f` over the `Interval` or
+`IntervalBox` `dom` using the Moore-Skelboe algorithm.
 
 We refer to the documentation and source code of `IntervalOptimisation` for details
 on the implemented methods.
 """
 function enclose_IntervalOptimisation(f::Function, dom::Interval_or_IntervalBox;
                                       structure=HeapedVector, tol=1e-3)
 
-    global_min, _, global_max, _ = range_and_optimisers(f, dom,
-                                                        structure=structure, tol=tol)
+    global_min, _ = minimise(X->f(X...), dom, structure=structure, tol=tol)
+    global_max, _ = maximise(X->f(X...), dom, structure=structure, tol=tol)
```

Comment:
```suggestion
    global_min, _ = minimise(enclose_IntervalArithmetic, dom, structure=structure, tol=tol)
    global_max, _ = maximise(enclose_IntervalArithmetic, dom, structure=structure, tol=tol)
```
This is a proposal based on [this other comment](https://github.com/JuliaReach/RangeEnclosures.jl/pull/69#discussion_r846651973). I am not sure if it is actually a good idea.

Reply:
in general I like the idea, because after the interface is more mature and we have different evaluation methods, one could choose if evaluating the function with natural enclosure (the current proposal), or mean value form, or taylor models etc. this would help overcome the [clustering problem](https://juliaintervals.github.io/pages/tutorials/tutorialOptimisation/#clustering_problem)

Since the API will change anyway, maybe this can be postponed? Or maybe done already here to remember the design structure.

Anyway, I think it should be `X->enclose_IntervalArithmetic(f, X)`

Reply:
> Anyway, I think it should be `X->enclose_IntervalArithmetic(f, X)`

Indeed!

> Since the API will change anyway, maybe this can be postponed?

Okay, I will merge then :+1: